adobecom/milo
# MWPW-142622: [Project PEP] Close button focus

## Description
This PR fixes the bug where the blue focus outline of the PEP X button does not show, when PEP is initiated after the region picker modal.

## Related Issue
Resolves: [MWPW-142622](https://jira.corp.adobe.com/browse/MWPW-142622)

## Testing instructions
1. Go to the test URL;
2. Sign in;
3. Wait for the region picker modal to show up and close it;
4. When PEP is initiated, the X button should be focused and have a blue outline;

## Screenshots (if appropriate):
![image](https://github.com/adobecom/milo/assets/146744221/dd8f1793-3166-4122-9c95-d89bcd628073)

## Test URLs
**Milo:**
- Before: https://project-pep--milo--adobecom.hlx.page/drafts/rbogos/pep-prompt?martech=off&imsClientId=fedsmilo
- After: https://mwpw-142622-project-pep--milo--robert-bogos.hlx.page/drafts/rbogos/pep-prompt?martech=off&imsClientId=fedsmilo
